README.md

```diff
@@ -414,21 +421,28 @@ cd client; go test  -tags sqlite  .
 
 We use [Cypress](https://www.cypress.io) to run our e2e tests. You can run all tests using:
 
-```shell script
+<pre type="make/command">
 make test-e2e
-```
+</pre>
 
 If you intend developing e2e tests, run the following command for more details:
 
-```shell script
+<pre type="repo/executable">
 ./test/e2e/run.sh
-```
-
+</pre>
 
 #### Build Docker
 
 You can build a development Docker Image using:
 
-```shell script
+<pre type="make/command">
 make docker
-```
+</pre>
+
+#### Documentation Tests
+
+To prepare documentation tests, run `npm i` to install
+[Text-Runner](https://github.com/kevgo/text-runner).
+
+- test all documentation: <code type="make/command">make test-docs</code>
+- test an individual file: <code type="npm/installed-executable">text-run</code>
```

Comment:
This block of type `npm/installed-executable` contains an executable installed by an external npm package. Text-Runner verifies it by looking for this file inside `node_modules/.bin`.